backend/package.json

```diff
@@ -41,8 +41,9 @@
   },
   "devDependencies": {
     "@types/compression": "^1.7.2",
-    "@types/ws": "~8.5.3",
     "@types/express": "^4.17.13",
+    "@types/ws": "~8.5.3",
+    "cross-env": "^7.0.3",
```

Comment:
@ayanamidev why is this dependency being added? It's no longer maintained and doesn't seem to be needed or in scope for this change

Reply:
@knorrium Seems like adding `node` would be sufficient to enable `npm run build` for Windows. Replacing this one.